src/luxonis_ml/data/augmentations.py

```diff
@@ -696,3 +697,72 @@ def get_params_dependent_on_targets(self, params: Dict[str, Any]) -> Dict[str, A
         target_params = super().get_params_dependent_on_targets(params)
         target_params["indices"] = list(range(self.n_tiles))
         return target_params
+
+
+@AUGMENTATIONS.register_module()
+class MixUp(A.BatchBasedTransform):
+    def __init__(
+        self,
+        alpha: float = 0.5,
```

Comment:
Could we add on option to randomize the `alpha` as well?
The `alpha` could be either:
- a single float - in that case it would be fixed
- a tuple of two floats - bounds for a uniform distribution.